source/tutorial/install-auth-plugin-c.txt

```diff
@@ -47,9 +47,9 @@ Connector for BI 2.2.
 Install ``mongosql_auth`` on Windows
 ------------------------------------
 
-#. Download the `MySQL 5.7.18 installer
-   <https://dev.mysql.com/downloads/file/?id=470091>`_ and install the
-   products needed.
```

Comment:
This needs to be less ambiguous. They need to download the MySQL Community Server that includes `mysql.exe`. There is a separate product called MySQL Shell (`mysqlsh.exe`) that we do not support. Just telling them to install Community Server should be enough.

Reply:
I wonder if we should go whole hog and explain that although they need to install the MySQL community server, they don't need to set it up as a service and start it? That seems like a lot of extra verbiage, but it would be a shame if anyone thought they needed to actually run the MySQL server in order to use BI. I'll leave it out for now, but it's something to think about.